keps/sig-cloud-provider/20190422-cloud-controller-manager-migration.md

```diff
@@ -127,7 +127,8 @@ The properties of the migration lock are:
   * no two migration locks should have overlapping controllers
   * the controller manager where the lock runs can change across releases.
   * for a minor release it should run exclusively in one type of controller manager - KCM or CCM.
-
+  * given a minor release, all subsequent patch releases shouldn't change the migration locks
```

Comment:
Tempted to suggest we fix the header for this list. Rather than properties, it might be better to phrase it as requirements or pre-conditions. These are not characteristics we are creating or enforcing. There are characteristics we are requiring.